main.tf

```diff
@@ -270,14 +273,14 @@ module "graphdb" {
   graphdb_outbound_address_prefixes    = var.outbound_allowed_address_prefixes
 
   # Gateway
-  application_gateway_backend_address_pool_ids = [module.application_gateway.gateway_backend_address_pool_id]
+  application_gateway_backend_address_pool_ids = var.disable_agw ? [] : [module.application_gateway[0].gateway_backend_address_pool_id]
 
   # App Configuration
   app_configuration_id       = module.appconfig.app_configuration_id
   app_configuration_endpoint = module.appconfig.app_configuration_endpoint
 
   # GraphDB Configurations
-  graphdb_external_address_fqdn = var.graphdb_external_address_fqdn != null ? var.graphdb_external_address_fqdn : module.application_gateway.public_ip_address_fqdn
+  graphdb_external_address_fqdn = var.graphdb_external_address_fqdn != null ? var.graphdb_external_address_fqdn : (var.disable_agw ? null : module.application_gateway[0].public_ip_address_fqdn)
```

Comment:
How does the `graphdb.properties` file look like when `disable_agw` is true?

Reply:
This comment is still valid

Reply:
When disabled_agw is true, I have added new validations (will push soon). It uses the Frontend public IP address of the external Application Gateway. It also adds the context path if there is one.

Reply:
Yes, external fdqn variable is needed. I will add a note that it is required when `disabled_agw` is used.

Reply:
I still want to see the outputted `graphdb.properties` file